components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -15,6 +16,34 @@ import { DBTeamMembership } from "./entity/db-team-membership";
 import { DBUser } from "./entity/db-user";
 import { DBTeamMembershipInvite } from "./entity/db-team-membership-invite";
 
+const FORBIDDEN_SLUGS = [
```

Comment:
**praise**: Thanks for extending the forbidden slugs here! 🌟

Reply:
How do we ensure this list does not drift w.r.t. the dashboard URLs?

Reply:
I don't think a perfect match is required at all times. The imported blocklist already contains a lot of teams names we wouldn't want to ever allow, and this complementary list ensures that current and potential near future dashboard paths aren't immediately taken by teams.

While a collision could still potentially happen in the more distant future, it's highly unlikely, and would be a sufficiently small problem that we can deal with it ad-hoc then.